py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -123,7 +126,8 @@ def __parse_config(self, config):
         self.raw_statistics_filename = config.get("raw statistics file")
 
         self.force_stack_delta_to_zero = config.getboolean("force stack delta to zero")
-
+        self.use_splines=config.getboolean("use splines")
```

Comment:
We should raise ExpectedFluxError when the different options are `None` if only to check that the default values are properly digested (it could not be the case if, for example, one is running from a notebook. I will add a commit to fix this